### DISCUSSION TITLE
"Upload proof of your academic status" page not working correctly

### DISCUSSION THREAD

**Discussion by sametkesici** [community/community#66694] [GitHub Education]
### Hi everyone,

hello  "https://education.github.com/discount_requests/xxxx/additional_information" this page not working correctly. The page asks for a photo of a document, but only the front camera opens on the phone. I tried on ios and android, it doesn't open the rear camera. Besides the photo taking option, why wasn't a button added where we can directly upload the document ?

**Comment by shreya-oeo** [comment]
Try doing it via pc or github website on desktop mode, you can upload the picture there.

**Reply by sametkesici** [reply]
i cant see anything for upload doc or picture. Only take button is here :(
<img width="1229" alt="Screen Shot 2023-09-12 at 02 01 09" src="https://github.com/community/community/assets/46839514/ecd11af9-9d39-4dd5-ac84-028b8855e139">

**Reply by gabrielqueeiroz** [reply]
I have the same issue

**Reply by Superskyyy** [reply]
same here

**Reply by Abhijeet-980** [reply]
same issue here
